demo/site/src/common/blocks/TextImageBlock.tsx

```diff
@@ -9,33 +11,41 @@ import { RichTextBlock } from "./RichTextBlock";
 export const TextImageBlock = withPreview(
     ({ data: { text, image, imageAspectRatio, imagePosition } }: PropsWithData<TextImageBlockData>) => {
         return (
-            <Root $imagePosition={imagePosition}>
-                <ImageContainer>
-                    <DamImageBlock data={image} aspectRatio={imageAspectRatio} sizes="50vw" />
-                </ImageContainer>
-                <TextContainer>
-                    <RichTextBlock data={text} />
-                </TextContainer>
-            </Root>
+            <PageLayout>
```

Comment:
By adding <PageLayout> into a block, you are converting it into a PageContent-Block (a block that positions itself in the page (adds margins etc)).

PageContent-Blocks can only be used in the outer PageContentBlock (the root content-BlocksBlock) as it would add extra margins for nested usages.

common/blocks/* blocks can have two exports: A PageContent version and a plain version.